### DISCUSSION TITLE
Apktool Batch Script

### DISCUSSION THREAD

**Discussion by ghostlium** [iBotPeaches/Apktool#2600] [Q&A]
Good day, or good afternoon to everyone here. I am wondering if it is possible to run Apktool through a batch script multiple times (such as a recursive script). I have tried to do this, but it appears that I am only able to run it the first initial time. If it is possible that someone can help me, please do.

**Comment by iBotPeaches** [comment]
Yes, unless I misunderstood. I have basic bash files that do things like:

```
for file in *.apk; do apktool d $file; done;
```

What do you mean it only runs initial time?

**Reply by ghostlium** [reply]
Never mind, it was likely due to an issue with my install since it started working when I reinstalled the files.

Is there any way to get rid of the "Press any key to continue . . ." when running through a batch script? I know it doesn't happen in bash, but I was wondering if I could just click the batch file and run it without that message.

**Comment by jie65535** [comment]
`echo "" | apktool ...`
